storybooknative/src/InputField/index.js.flow

```diff
@@ -1,24 +1,20 @@
 // @flow
 
-import * as React from "react";
-
-import Text from "../Text";
-
 export type Props = {|
   +size?: "small" | "normal",
   +type?: "text" | "number" | "email" | "password",
   +defaultValue?: string,
   +autoFocus?: boolean,
   +keyboardType?: "email-address" | "numeric" | "number-pad",
   +secureTextEntry?: boolean,
-  +label?: React.Element<typeof Text>,
+  +label?: string,
```

Comment:
These are props that the consumer will pass in, this will not work with our setup, since we use translation component. It should also work for components. 

Same goes for help and error I guess.